idl/storage_service.idl.hh

```diff
@@ -59,8 +59,17 @@ struct raft_topology_snapshot {
 
 struct raft_topology_pull_params {};
 
+struct raft_snapshot {
+    utils::chunked_vector<canonical_mutation> mutations;
+};
+
+struct raft_snapshot_pull_params {
+    std::vector<table_id> tables;
```

Comment:
This seems suspicious to me, are you sure those `table_id`s are consistent across different nodes? Because e.g. `raft_pull_topology_snapshot` handler looks up the table based on the names (keyspace and table names), not table ID. What guarantees that table ID will be the same here and remotely? After all, these are local tables, generated separately by each node

Reply:
I think it's the same due to:
```
table_id
generate_legacy_id(const sstring& ks_name, const sstring& cf_name) {
    return table_id(utils::UUID_gen::get_name_UUID(ks_name + cf_name));
}
```